lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -2,18 +2,22 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <%= cond do %>
-    <%= page.id == :module -> %>
-    <title><%= page.content %></title>
-    <%= page.id == :readme -> %>
-    <title><%= page.content.project %> v<%= page.content.version %> README</title>
-    <%= page.id == :overview -> %>
-    <title><%= page.content.project %> v<%= page.content.version %> Overview</title>
+      <%= page.id == :module -> %>
+        <%= if page.module == config.project do %>
+          <title><%= config.project %> v<%= config.version %> – Documentation</title>
+        <% else %>
+          <title><%= page.module %> – <%= config.project %> v<%= config.version %></title>
+        <% end %>
+      <%= page.id == :readme -> %>
+         <title>README – <%= config.project %> v<%= config.version %></title>
+      <%= page.id == :overview -> %>
+        <title>Overview – <%= config.project %> v<%= config.version %></title>
     <% end %>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="generator" content="ExDoc">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
`meta` is a void element, according to the HTML5 spec the `/` character has no effect.